course/models.py

```diff
@@ -146,6 +160,16 @@ def get_student_courses(self, id: int):
     def get_student_confirmed(self, id: User):
         return self.filter(student_id=id, status=Status.CONFIRMED)
 
+    def is_student_enrolled_in_course(self, course: TeacherCourse, student: User):
+        if self.filter(teacher_course_id=course, student_id=student, status=Status.CONFIRMED):
```

Comment:
instead of returning `True` or `False`, you can return `bool(self.filter(teacher_course_id=course, student_id=student, status=Status.CONFIRMED))`